.wiki/Configuration.md

```diff
@@ -37,6 +42,47 @@ for [repository definitions](https://github.com/artipie/artipie/wiki/Configurati
 It locates a storage where all config files for each repository are located. Keep in mind,
 Artipie user should have read and write permissions for this storage.
 
-You may want configure it via environment variables:
+Here is the example of Artipie configuration files structure for `flat` layout based on 
+`/etc/artipie/artipie.yml` example above:
+```
+/tmp/artipie/configs
+│   _storages.yaml
+│   _credentials.yaml    
+│
+└───repo
+│   │   my-maven.yml
+│   │   main-python.yaml
+│   │   docker-registry.yaml
+```
+
+If the layout is `org`, then repository configurations will be located in user's subdirectories:
```

Comment:
> will be located in **user's**

Where is `user` from? 
We didn't use the word `user` anywhere above, it's confusing.
We used the word `organization` when we were talking about layouts above.